udata_front/theme/gouvfr/templates/base.html

```diff
@@ -14,7 +14,9 @@
     {% endif %}
 </div>
 {% endwith %}
-{% block content %}{% endblock %}
+<main>
```

Comment:
We don't need this tag for accessibility right?

Reply:
`<main>` or a tag with `role="main"` is required on each page